rosidl_generator_c/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ add_library(${PROJECT_NAME} SHARED
 )
 if(NOT WIN32)
   set_target_properties(${PROJECT_NAME} PROPERTIES
-    COMPILE_FLAGS "-std=c11 -Wall -Wextra")
+    COMPILE_FLAGS "-Wall -Wextra")
```

Comment:
I disagree, we should add it when we need it. It has received a lot less adoption than C++11 and so it isn't as low risk to make a hard dependency on it.